gammapy/background/fov_cube.py

```diff
@@ -708,6 +710,59 @@ def make_spectrum(self, coord, ebounds=None):
 
         return spectrum
 
+    def make_roi_spectrum(self, offset, roi, ebounds=None):
+        """
+        Generate energy spectrum around a certain position in the FOV within an integration radius
+
+        Parameters
+        ----------
+        offset : `~astropy.units.Quantity`
+            Offset of the integration area
+        roi : `~astropy.units.Quantity`
```

Comment:
I am not 100% sure, but it seems like one could use [astropy.regions](https://github.com/astropy/regions) for this. I see t hat further down you compute containment for a circle which is basically what the [masking functionality](http://astropy-regions.readthedocs.io/en/latest/masks.html) of the ``regions`` package does. Do you know the ``regions`` package and did you consider using it here?

Reply:
Indeed, this implementation is the 0st order algorithm, ie using a circle only. It can be generalised..
Can you help me for that? ie remove the RoI by a region? Is is compatible with the new implementation of maps?

Reply:
I think this is a method on the "old" FOV background cube class? Suggest to not spend time on it like @joleroi suggested, it will probably go away. Like I said today, I think `gammapy.maps` (and new IRF classes following what @joleroi started a year ago, will have to see how those fit together).

I think regions is only really needed for sky regions, we don't have / need a region type in FOV coordinates. So suggest to just leave the circle here for now, and then to look at if regions are used in `gammapy.maps` already, or if it would make sense to add methods that use region objects as input there (like the one or two we already have on SkyImage / SkyCube).